lib/screens/prescriptions.list.screen.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+
+import '../utils/color_schemes.g.dart';
+import '../widgets/prescription.item.dart';
+import '../widgets/styles/KTextStyle.dart';
+
+class PrescriptionsListScreen extends StatefulWidget {
+  const PrescriptionsListScreen({super.key});
+
+  @override
+  State<PrescriptionsListScreen> createState() =>
+      _PrescriptionsListScreenState();
+}
+
+@override
+void initState() {
```

Comment:
Here comes, my place I want to print data here in console by calling `PersonRepository.getAllPatients()` method

**Steps to reach on this screen**
from `onBoarding screen` Tap on `patient card` then tab `Verify button` finally that list of prescriptions screen
![image](https://github.com/nahimanajz/Healthier/assets/50912668/0f04da48-5d28-40a1-92ca-fc0e80a6a72a)
![image](https://github.com/nahimanajz/Healthier/assets/50912668/38c625b5-dd4c-47fe-88c7-5869a2c8b9d9)
![image](https://github.com/nahimanajz/Healthier/assets/50912668/2b711ea9-c43d-40f6-a67f-96666ad5c3a5)